### ISSUE TITLE
Argument list too long

### ISSUE THREAD

**Issue by jimko** [GhostWriters/DockSTARTer#1338]
# Bug report

**Describe the bug**
Error reported while running ds -c. Happens with or without a docker-compose.override.yml file present.
/usr/local/bin/ds: line 75: /usr/bin/sudo: Argument list too long

**To Reproduce**
Run a full setup and select handbrake, homeassistant, watchtower. (not sure if these 3 are essential in reproducing, but that is what I have started with)
cd 
ds -c

**Expected behavior**
Line 75 doesn't look like it's doing anything critical, but probably shouldn't spit a warning

**Screenshots**
```
jimko@myhost:~$ ds -c
2021-06-20 16:22:17 [WARN  ]   Setting permissions for /srv/dev-disk-by-uuid-e1f966d0-9af5-41eb-93d9-7660cb96d1f6/appdata/dockstarter/.config/.compose.backups outside of /srv/dev-disk-by-uuid-e1f966d0-9af5-41eb-93d9-7660cb96d1f6/one/jimko may be unsafe.
2021-06-20 16:22:17 [NOTICE]   Creating environment variables for enabled apps. Please be patient, this can take a while.
2021-06-20 16:22:18 [NOTICE]   Adding compose configurations for enabled apps. Please be patient, this can take a while.
latest: Pulling from linuxserver/yq
Digest: sha256:c892a90eb3df0dcde6ead024eed0ff24830b5dc77dbb65685b6d2ef05ab68351
Status: Image is up to date for ghcr.io/linuxserver/yq:latest
ghcr.io/linuxserver/yq:latest
latest: Pulling from linuxserver/docker-compose
Digest: sha256:57d36dd05592ac6e7763f470a91394d585fd33c043a53dda40b0480681d8ee66
Status: Image is up to date for ghcr.io/linuxserver/docker-compose:latest
ghcr.io/linuxserver/docker-compose:latest
handbrake is up-to-date
homeassistant is up-to-date
watchtower is up-to-date
/srv/dev-disk-by-uuid-e1f966d0-9af5-41eb-93d9-7660cb96d1f6/one/jimko/.docker/main.sh: line 75: /usr/bin/sudo: Argument list too long
jimko@myhost:~$ 
```

**System**
Open Media Vault
```
uname -r
5.10.0-0.bpo.7-amd64

cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

**Additional context**
Minor, no apparent loss of functionality.

**Comment by macr0dev** [comment]
I came in here looking for a solution to this problem, and found your issue.  After doing some experimentation and having some other odd problems - I think it may be directly related to having "too many" containers setup.  I had been trying to add two (tdarr/tdarrnode) and was getting this error.  I dropped two other containers that I ended up not using but still had and everything started working just fine.

So I'm not sure what the solution here is.  I'm currently running 21 containers.  You?
